PendingReleaseNotes

```diff
@@ -13,6 +13,14 @@
   output is changed from 'STATUS' to 'STATE'. The state of a group snapshot
   that was shown as 'ok' is now shown as 'complete', which is more descriptive.
 
+>=19.2.1
```

Comment:
We already have `>=19.0.0`, we don't need this.

Reply:
But this feature is available starting 19.2.1 only. I see specific versions are mentioned in release notes already for other features as well https://github.com/ceph/ceph/blob/main/PendingReleaseNotes#L510